### DISCUSSION TITLE
Any recommendation where i can deploy / host goTrue?

### DISCUSSION THREAD

**Discussion by goldcoders** [supabase/supabase#689] [Questions]
I wanna try to deploy goTrue,

What hosting provider / cloud provider 

is best you guys can recommend

**Comment by kiwicopple** [comment]
Hey @goldcoders - it should work with basically any hosting provider that supports either Go or Docker (eg: all of them)

So my recommendation is to use whichever you are most familiar with.

Do note that our fork is slightly altered from Netlify's: https://github.com/netlify/gotrue. You could also deploy theirs if you want to use it backed by a MySQL database
